utils/manage_static.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
+# -*- coding: utf-8; mode: sh indent-tabs-mode: nil -*-
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+BUILD_COMMIT_MESSAGE="Static build"
+
+BUILT_PATHS=(
+    searx/static/themes/oscar/css
+    searx/static/themes/oscar/js
+    searx/static/themes/oscar/src/generated/pygments-logicodev.less
+    searx/static/themes/oscar/src/generated/pygments-pointhi.less
+    searx/static/themes/simple/css
+    searx/static/themes/simple/js
+    searx/static/themes/simple/src/generated/pygments.less
+)
+
+CURRENT_BRANCH="$(git branch --show-current)"
+STAGED_FILES=$(git diff --name-only --cached)
+
```

Comment:
I see drawbacks with this globlas:

1. git is called when the script is loaded
2. the values are inited when the script is loaded, but the script itself changes the origin / I mean after a static.git.restore the values in this globals are old.

The last one is not a problem in this small script, but when we consider to integrate this script as a library there might be use cases where we e.g. call static.build.commit.drop twice in the same shell process.

I will commit some patches on top of yours ... coming soon.